src/UserManagerSettings.ts

```diff
@@ -192,4 +195,13 @@ export class UserManagerSettingsStore extends OidcClientSettingsStore {
             this.userStore = new WebStorageStateStore({ store });
         }
     }
+
+    private validatePopupWindowFeatures(popupWindowFeatures: PopupWindowFeatures): void {
+        if (popupWindowFeatures.closeAutomaticallyPopupWindow) {
+            if (popupWindowFeatures.closePopupWindowAfter !== undefined && popupWindowFeatures.closePopupWindowAfter !== null && popupWindowFeatures.closePopupWindowAfter <= 0) {
+                console.log("validatePopupWindowFeatures");
```

Comment:
please avoid this kind of logs, iff usful logs please use the logging system

Reply:
if only using `closePopupWindowAfter`, we anyway do not need this `validatePopupWindowFeatures` function...

Reply:
Oh sorry there shouldn't be console. Ok I fixed it